yoti_test.go

```diff
@@ -904,12 +904,12 @@ func TestAnchorParser_Passport(t *testing.T) {
 		t.Error("Anchors and Sources should be the same when there is only one Source")
 	}
 
-	if actualAnchor.Type != anchor.AnchorTypeSource {
-		t.Errorf("Parsed anchor type is incorrect. Expected: %q, actual: %q", anchor.AnchorTypeSource, actualAnchor.Type)
+	if actualAnchor.Type() != anchor.AnchorTypeSource {
+		t.Errorf("Parsed anchor type is incorrect. Expected: %q, actual: %q", anchor.AnchorTypeSource, actualAnchor.Type())
```

Comment:
Consider splitting long lines, currently have to scroll sideways on GitHub to view the whole line in quite a few places.